src/libasr/pass/intrinsic_functions.h

```diff
@@ -2075,10 +2075,14 @@ namespace Dim {
 namespace Sqrt {
 
     static ASR::expr_t *eval_Sqrt(Allocator &al, const Location &loc,
-            ASR::ttype_t* arg_type, Vec<ASR::expr_t*> &args, diag::Diagnostics& /*diag*/) {
+            ASR::ttype_t* arg_type, Vec<ASR::expr_t*> &args, diag::Diagnostics& diag) {
         ASRUtils::ASRBuilder b(al, loc);
         if (is_real(*arg_type)) {
             double val = ASR::down_cast<ASR::RealConstant_t>(expr_value(args[0]))->m_r;
+            if (val < 0.0) {
+                append_error(diag, "Argument of SQRT has a negative value", loc);
```

Comment:
I prefer using lowercase:
```suggestion
                append_error(diag, "Argument of `sqrt` has a negative value", loc);
```
The uppercase reminds me of old F77 style, while lowercase is how most modern languages are used.